client/karma.config.js

```diff
@@ -105,6 +106,9 @@ module.exports = function(karma) {
         ]
       },
       plugins: [
+        new webpack.ProvidePlugin({
+          process: 'process/browser',
```

Comment:
What do we need this for and can't we get rid of it with `DefinePlugin`?

Reply:
Webpack@5 no longer provides nodeJS polyfills for browsers by default. In the tests, we use [`process.nextTick()` ](https://github.com/camunda/camunda-modeler/blob/58123a652018db7d73ce8274d98713eece3f6020/client/src/app/__tests__/AppParentSpec.js#L466), which now needs to be provided by this plugin.

The Define plugin is still needed so we can supply env variables

Reply:
Uhm... let's get rid of `process.nextTick?`